geometry_msgs/Point object_position
---
uint8 grasping_status
